lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -377,12 +528,14 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
 
         if (defined(this.layers)) {
             var layers = this.layers.split(',');
+            var styles = (this.styles || this.parameters.styles || '').split(',');
```

Comment:
Yep, exactly.  I'm not totally convinced it's the right thing, but consistency is good... right?